docs/arf.md

```diff
@@ -2393,6 +2453,7 @@ The technical implementation of this verification depends on which of
 the standards mentioned in \[Topic 12\] is supported by the Wallet
 Instance. Each of these standards specifies in detail how to carry out
 this verification.
+[**Can you expand on what happens when a "cloud wallet" is being used and there could be multiple devices that access the cloud wallet?**]
```

Comment:
From the point of view of the Relying Party, the internal architecture of the Wallet Unit does not have any impact.  In general, the ARF does not discuss the various architectures that could be used to implement a Wallet Unit. The only requirement is that it must be possible to certify each solution.